packages/subgraph/src/subgraph.ts

```diff
@@ -30,19 +55,40 @@ export default class Subgraph {
      * @param options Options to select the group parameters.
      * @returns List of groups.
      */
-    async getGroups(options: GroupOptions = {}): Promise<any[]> {
+    async getGroups(options: GroupOptions = {}): Promise<GetGroupsResponse[]> {
```

Comment:
Should we add the `GetGroupsResponse` return type to the `getGroup` function too?

Reply:
Ah yes, missed that. Added now.